drivers/volume/portworx/portworx.go

```diff
@@ -3956,4 +4068,11 @@ func init() {
 	if err := storkvolume.Register(storkvolume.PortworxDriverName, p); err != nil {
 		logrus.Panicf("Error registering portworx volume driver: %v", err)
 	}
+
+	soPath := path.Join(statfsSODirInStork, statfsSOName)
+	statfsSOContents, err = ioutil.ReadFile(soPath)
```

Comment:
Why read this in as a global var, keeping it in memory the whole time instead of just when we need it?

Reply:
Didn't want to read the same file over and over unnecessarily. Do you see a problem here?

Reply:
Are kubevirt pods common enough for us to keep in in memory for every single instance of the portworx driver? If someone is using them do they start new pods very frequently? If statfsSOContents is small I guess it's not a big deal, but I'm suspicious of the pattern. Reading a file occasionally isn't a big deal, but ballooning memory footprint could be.

Reply:
Yeah, I see your point. Will change it.